src/components/interview.ts

```diff
@@ -3,14 +3,25 @@ import Discord from 'discord.js';
 import _ from 'lodash';
 
 const RESULTS_PER_PAGE = 6;
-
+const available_domains: { [key: string]: string } = {
+  frontend: 'Frontend',
+  backend: 'Backend',
+  design: 'Design',
+  pm: 'PM'
+};
 interface Interviewer {
-  user_id: number;
+  user_id: string;
   link: string;
 }
 
+function domainString() {
```

Comment:
this function could take a `domains` parameter, so it's more reusable

Reply:
nit: rename to `getDomainsString`, a function should preferably start with a verb/action word

Reply:
in this case arrow functions might look cleaner
```
const getDomainsString = domains => _.join(domains, ', ');
```
I kinda think we should use arrow functions for the other functions too just to be consistent.